tutorials/setup_tutorial.md

```diff
@@ -38,11 +38,22 @@ before running any `jiant` code. (To deactivate run: `source deactivate`)
 
 Some requirements may only be needed for specific configurations. If you have trouble installing a specific dependency and suspect that it isn't needed for your use case, create an issue or a pull request, and we'll help you get by without it.
 
-You will also need to install dependencies for `nltk` if you do not already have them:
+You will also need to install dependencies for `nltk` and `spacy' if you do not already have them:
```

Comment:
Mention where SpaCy is needed. It's not part of the main codeflow for lots of models, so some users can skip this.

Reply:
Done.